### PR TITLE
fix: [CDS-50824]: execution testing bugs

### PR DESCRIPTION
**Pull request by TarunUba1**
## Describe your changes
setting execution id for shell script to be equal to servicehook id and action. Editing error message
## Testing
Screenshot 2023-05-24 at 11 24 50 AM
Screenshot 2023-05-24 at 11 24 57 AM

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/48276)

### CODE REVIEW

**Comment by TarunUba1** [comment]
triggere uts

File: 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java [hunk 1]
```diff
@@ -137,27 +135,24 @@ private ExecuteCommandResponse executeCommand(
       ScriptProcessExecutor processExecutor, String errorMessage, String directoryPath, String script) {
     try {
       createDirectoryIfDoesNotExist(Paths.get(String.valueOf(directoryPath)));
-      final ExecuteCommandResponse executeCommandResponse =
-          processExecutor.executeCommandString(script, emptyList(), Collections.emptyList(), null, false);
-      return executeCommandResponse;
+      return processExecutor.executeCommandString(script, emptyList(), Collections.emptyList(), null, false);
```
Comment (abhinavsingh1196) [inline 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java:138]: emptyList() and Collections.emptyList() both being used here

**Comment by TarunUba1** [comment]
trigger uts

File: 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java [hunk 2]
```diff
@@ -199,10 +194,6 @@ List<ServiceHookDelegateConfig> requiredHooks(String action, String type) {
         .collect(Collectors.toList());
   }
 
-  private void saveExecutionLog(String line, LogLevel level, LogCallback logCallback) {
```
Comment (abhinavsingh1196) [inline 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java:202]: unused method?

**Review by abhinavsingh1196 — CHANGES_REQUESTED** [review]

**Review by abosii-harness — DISMISSED** [review]

File: 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java (hunk 2 above)
Comment (TarunUba1) [reply 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java:202]: yes

File: 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java (hunk 1 above)
Comment (TarunUba1) [reply 930-delegate-tasks/src/main/java/io/harness/delegate/utils/ServiceHookHandler.java:136]: aah , will change it

**Comment by TarunUba1** [comment]
trigger uts

**Review by abhinavsingh1196 — APPROVED** [review]

**Review by alebuh — APPROVED** [review]

**Comment by abhinavsingh1196** [comment]
trigger review

**Review by namantalaycha10 — APPROVED** [review]

**Pull request merged**